tools/bundle_adjust_tracks.cxx

```diff
@@ -1023,16 +1025,22 @@ static int maptk_main(int argc, char const* argv[])
   //
   if (config->get_value<bool>("krtd_clean_up") )
   {
-    std::cout << "Cleaning " << config->get_value<std::string>("output_krtd_dir") << " before writing new files." << std::endl;
 
-    std::ifstream framelist(config->get_value<std::string>("image_list_file"));
-    std::string filePath, krtdFile;
+    std::cerr << "Cleaning "
+              << config->get_value<std::string>("output_krtd_dir")
+              << " before writing new files." << std::endl;
 
-    while (framelist >> filePath)
-    {
-      krtdFile = ST::GetFilenameWithoutExtension(filePath) + ".krtd";
+    boost::filesystem::path p(config->get_value<std::string>("output_krtd_dir"));
+    boost::filesystem::directory_iterator end_iter;
 
-      ST::RemoveFile(config->get_value<std::string>("output_krtd_dir") + krtdFile);
+    for( boost::filesystem::directory_iterator dir_iter(p);
```

Comment:
We don't want to pull Boost in for this.  We are trying to avoid needing to depend on Boost.  In this case it's easy.  There is already a function "files_in_dir()" implemented in this file around line 365.  You can just call that to get a list of files in a directory.

We are avoiding Boost by using [KWSys](https://github.com/commontk/KWSys).  There is a copy of KWSys in Vital named kwiversys.  You can do most of the same file path manipulations with that instead of Boost.